src/server/debug.rs

```diff
@@ -266,6 +266,43 @@ impl Debugger {
         }
     }
 
+    pub fn do_unsafe_recover(&self, region_id: u64) -> Result<()> {
```

Comment:
If we encounter this problem, then it's not just a problem of a region but rater the whole store. So instead of taking a region id, it should take a store id which is in unrecoverable state, and then recover all existing affected regions on this nodes.

Reply:
yes, it's better.